lib/statsd/instrument/metric_expectation.rb

```diff
@@ -0,0 +1,30 @@
+class StatsD::Instrument::MetricExpectation < StatsD::Instrument::Metric
+
+  attr_accessor :times, :ignore_tags
+
+  # Initializes a new MetricExpectation instance.
+  #
+  # @times times [Numeric] :times The number of times this metric is expected
+  # @@ignore_tags @ignore_tags [Array<String>, Hash<String, String>, nil] :@ignore_tags The tags
+  #   to ignore for this metric.
+  def initialize(options = {})
```

Comment:
Does the rest of this codebase use options hashes? Because if not then I would suggest using keyword arguments instead.